Formula/zork.rb

```diff
@@ -0,0 +1,30 @@
+class Zork < Formula
+  desc "The Dungeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/v1.0.0.tar.gz"
+  sha256 "ceedb436cf552bd8ca39abd3e85bae986c3f5df6f31c1bf2806d1b4c8509da67"
+  head "https://github.com/devshane/zork.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make", "LIBDIR=#{libexec}", "BINDIR=#{bin}"
+    bin.mkpath
+    libexec.mkpath
+    system "make", "install", "LIBDIR=#{libexec}", "BINDIR=#{bin}"
+    man6.install "dungeon.6"
+  end
+
+  test do
+    test_phrase = "Welcome to Dungeon.\t\t\tThis version created 11-MAR-91.\n"\
+        "You are in an open field west of a big white house with a boarded\n"\
+        "front door.\nThere is a small mailbox here.\n"\
+        ">Opening the mailbox reveals:\n  A leaflet.\n>"
+    require "open3"
+    Open3.popen3("#{bin}/zork") do |stdin, stdout, _|
```

Comment:
Oh neat, I didn't notice `pipe_output`, gave it a go and it looks like it works, pushed the change. 

```
❯ brew test --debug zork
/opt/brew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /opt/brew/Library/Taps/homebrew/homebrew-core/Formula/zork.rb
Testing zork
==> Using the sandbox
/opt/brew/Library/Homebrew/test.rb (Formulary::FromPathLoader): loading /opt/brew/Library/Taps/homebrew/homebrew-core/Formula/zork.rb
==> /opt/brew/Cellar/zork/1.0.1/bin/zork
The game is over.
```